Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static string NameofPerson()
```

Comment:
Functions should be verbs. You should specify an action in name. Get/Prompt/Read, ... and only then what- NameOfPerson.

Reply:
I will do it right away.

Sincerely,
Memoire